backend/sql/drop.sql

```diff
@@ -1,7 +1,10 @@
 drop table if exists project cascade;
```

Comment:
perhaps it's good pr to change `drop.sql` to `drop schema public` :putya:

Reply:
That's another issue ([EDNA-32](https://issues.serokell.io/issue/EDNA-32)), maybe someone will do it one day :agaaga: